protos/flyteidl/admin/matchable_resource.proto

```diff
@@ -44,6 +47,11 @@ message ExecutionQueueAttributes {
   repeated string tags           = 1; 
 }
 
+message ExecutionCluster {
```

Comment:
calling this a cluster doesn't seem right to me

Reply:
We discussed offline, and agreed on ExecutionCluster.
cc @surindersinghp